docs/make.jl

```diff
@@ -11,8 +11,9 @@ let
     exampledir = joinpath(@__DIR__, "..", "examples")
     excludedirs = String[]
     excludefiles = String[]
-    if VERSION < v"1.1.0"
-        push!(excludefiles, "Symbolic double pendulum.jl")
+    if VERSION >= v"1.6"
+        push!(excludefiles, "6. Symbolics.jl")
+        push!(excludefiles, "7. Rigorous error bounds using IntervalArithmetic.jl")
```

Comment:
This is a bit annoying... I have already spent a lot of time on this PR, but I haven't been able to fix the issues with these two notebooks.
- For `6. Symbolics.jl`, I am not sure how to fix it and there is an open issue for this (https://github.com/JuliaRobotics/RigidBodyDynamics.jl/issues/630).
- For `7. Rigorous error bounds using IntervalArithmetic.jl` I have already applied a few updates to use the new syntax and API of _IntervalArithmetic.jl_, but not all the cells of the notebook are fixed yet.